File: Java/q1060.java
// Números Positivos

import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) {
 
         Scanner input = new Scanner(System.in);
         int pos = 0;
         
         for(int i=0; i<6; i++){
             double n = input.nextDouble();
             if(n > 0){
                 pos++;
             }
         }
         
         System.out.printf("%d valores positivos%n", pos);
    }
 
}

File: Cpp/q1060.cpp
#include <iostream>
using namespace std;

int main(){

  int q;
  q = 0;

for(int i=0; i < 6; i++){
  float valor;
  cin >> valor;
  if(valor > 0){
      q++;
  }
}

  cout << q << " valores positivos\n";
  return 0;
}